components/main-nav.tsx

```diff
@@ -15,20 +15,22 @@ export function MainNav() {
         <span className='hidden font-bold sm:inline-block'>Ada URL</span>
       </Link>
       <nav className='flex items-center space-x-6 text-sm font-medium'>
+        {/*@TODO: add localized link*/}
         <Link
-          href='/docs'
+          href='en/docs/introduction'
```

Comment:
Link component receives a locale property

Reply:
For moment I use `LocalizedLink` because middleware isn't working